.github/workflows/deploy-preview.yml

```diff
@@ -7,17 +7,31 @@ on:
   pull_request:
```

Comment:
followed this guide https://hugo.alliau.me/2021/05/04/migration-to-github-native-dependabot-solutions-for-auto-merge-and-action-secrets/#share-your-secrets-with-dependabot to share secrets with dependabot via `pull_request_target`